src/main/kotlin/com/londogard/textgen/languagemodels/LanguageModel.kt

```diff
@@ -0,0 +1,87 @@
+package com.londogard.textgen.languagemodels
+
+import com.londogard.textgen.NGram
+import smile.nlp.tokenizer.SimpleTokenizer
+import smile.nlp.tokenizer.Tokenizer
+import kotlin.system.measureTimeMillis
+
+internal typealias InternalMutableLanguageModel = HashMap<List<Int>, MutableMap<Int, Double>>
+typealias InternalLanguageModel = Map<List<Int>, Map<Int, Double>>
+
+class LanguageModel2(private val tokenizer: Tokenizer = SimpleTokenizer(), val n: Int) {
```

Comment:
Why is it called LanguageModel2? With an emphasis on the 2

Reply:
Good comment, it's because of a clash with the old one. Gonna remove that one :)